src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -114,9 +122,13 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
   );
 
   const schedule: ScheduleDay = useMemo(() => {
-    const dayStart = addDays(startOfToday(), selectedDayIndex);
+    const dayStart = addDays(today, selectedDayIndex);
```

Comment:
It seems like we probably also want to use `firstDayOfSchedule` here as well. So it would make sense to pull it out of the `useMemo` above into its own `useMemo` block so it can easily be used in both places without code duplication/causing re-renders.